src/capi/azext_capi/custom.py

```diff
@@ -122,7 +93,7 @@ def _create_azure_identity_secret(cmd):
             output = subprocess.check_output(command, universal_newlines=True, stderr=stderr)
             logger.info("%s returned:\n%s", " ".join(command), output)
         except subprocess.CalledProcessError as err:
-            raise UnclassifiedUserFault("Can't create Cluster Identity Secret") from err
+            raise UnclassifiedUserFault(f"Can't create Cluster Identity Secret \n{err.stdout}") from err
```

Comment:
is the line return before printing the error intentional?

Reply:
I am not sure I am following correctly; do you mind elaborating? But if you are referring to `err.stdout` subprocess.CalledProcessError passes the error message of the command that it ran as an attribute and we are using it to be more specific of the type of error that it encountered.
Example:
The Cluster Identity Secret could fail to be created if there is an existing secrete with same required attributes.
Instead of just saying
```
Can't create Cluster Identity Secret
````
it will raise something along the lines
``` 
Can't create Cluster Identity Secret
An existing CIS with attribute already exists
```